components/common/LinkWraper.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import cn from 'classnames';
+
+import 'styles/src/kit/link/link.scss';
+
+export default ({ className, children, ...props }) => (
+  <a className={cn('wir-link', className)} {...props}>
+    {children}
```

Comment:
I think we need to wrap `{children}` here with `<span></span>`
Check this out: https://github.com/babajka/babajka-markup/blob/develop/src/kit/link/link.ejs#L6

Also: what if `children` is not text? In markup it's explicitly marked as text